calcom/cal.com
# Fixing active vs existing apps in event type

## What does this PR do?

Fixing active vs existing apps in event type
Plus strange margin in overlay and App Card.

Fixes #4221

**Environment**: Staging(main branch) / Production

## Type of change




## How should this be tested?

Go to Event Type details page in v2 and see the correct number of active apps versus installed apps available in submenu.
Also, when opening up Giphy dialog, a strange margin appeared in the AppCard and in the overlay, both fixed, see:

<kbd><img src="https://user-images.githubusercontent.com/467258/189763591-49c725f6-ec84-4d0a-b152-9724641c5b8a.png" /></kbd>

<kbd><img width="1503" alt="image" src="https://user-images.githubusercontent.com/467258/189764206-8766d052-231c-4a51-99a0-6b21abf9dad2.png"></kbd>
